README.md

```diff
@@ -29,23 +29,145 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
 
 ## Motivation
 
-* Users need all types of duration format depending on the requirements of their application. For example, in the airlines websites, to show how long a flight takes, the duration should be in Short or Narrow format
+* Users need all types of duration format depending on the requirements of their application. For example, to show how long a flight takes, the duration should be in Short or Narrow format
   - 1 hr 40 min 60 sec → Short 
   - 1 h 40 m 60 sec  → Narrow
 
-* Furthermore, for TTS (text to speech) applications, Wide format is needed
-  - 1 hour, 40 minutes and 60 seconds.
-
+## Requirements & Design
+
+In this section, we are going to illustrate each user needs (requirements) and a design for each need (requirement)
+
+### Input Value
+  * Users needs to identify the input values
```

Comment:
I'm not sure I understand what "identifying" input values entails. Perhaps a longer description would help here?

Reply:
okay, I added more descriptions